pyproject.toml

```diff
@@ -21,12 +21,13 @@ classifiers = [
 
 dependencies = [
     "rtslib-fb",
-    "configshell-fb"
+    "configshell-fb",
+    "PyGObject",
```

Comment:
understand it is used for dbus communication with the daemon.

Reply:
> perhaps because we don't support hatch very well yet? 

just a side-note, hatch doesn't need to be used at all. In PEP-517, you specify a build backend, which are then interoperable, so while you can do `hatch build`, you can also do `python -m build`. Same with pip install, etc. 

I'm looking at Leap, and not seeing hatch_vcs being available. If you want to build an rpm there while keeping the git-based versioning, you could change the pyproject.toml to:
```
[build-system]
requires = ["setuptools>=61", "setuptools-scm"]
build-backend = "setuptools.build_meta"
```
..or just remove hatch-vcs and do the versioning manually, as python311-hatchling is available :)